engine/src/flutter/lib/web_ui/lib/src/engine/semantics/tappable.dart

```diff
@@ -29,6 +29,13 @@ class SemanticButton extends SemanticRole {
     } else {
       removeAttribute('aria-disabled');
     }
+
+    // This is especially useful when the button is used to open a menu.
+    if (semanticsObject.hasExpandedState) {
```

Comment:
expansion tile can be a button, too.
Also this role already has Expandable behavior which will assign aria-expanded.

Reply:
Make sense! Reverted!

Reply:
Created an issue for the popup flag: https://github.com/flutter/flutter/issues/165505 :)